housekeeping-tool-parent/README.md

```diff
@@ -0,0 +1,88 @@
+# Housekeeping Tool
```

Comment:
I don't think we need this extra level here - let's go for:

root -> housekeeping-vacuum-tool - will produce a jar and a tgz (tool)
       -> housekeeping-core - will produce a jar file (library)